src/crimson/os/seastore/segment_cleaner.cc

```diff
@@ -423,6 +427,41 @@ SegmentCleaner::gc_reclaim_space_ret SegmentCleaner::gc_reclaim_space()
                   "SegmentCleaner::gc_reclaim_space: addr {} alive, gc'ing {}",
                   addr,
                   *ext);
+		assert(commit_time);
+		assert(commit_type == record_commit_type_t::MODIFY
+		  || commit_type == record_commit_type_t::REWRITE);
+		if (ext->get_last_modified() ==
+		    seastar::lowres_system_clock::time_point()) {
+		  assert(ext->get_last_rewritten() ==
+		    seastar::lowres_system_clock::time_point());
+		  ext->set_last_modified(
+		    seastar::lowres_system_clock::duration(
+		      commit_time));
+		}
+		if (ext->get_last_modified() ==
+		    seastar::lowres_system_clock::time_point()) {
+		  assert(ext->get_last_rewritten() ==
+		    seastar::lowres_system_clock::time_point());
+		  ext->set_last_modified(
+		    seastar::lowres_system_clock::duration(
+		      commit_time));
+		}
```

Comment:
Is this "if section" duplicated with the above "if section"?

Reply:
ah, yes.